src/tools/tool-options.h

```diff
@@ -82,7 +82,15 @@ struct ToolOptions : public Options {
       .addFeature(FeatureSet::MutableGlobals, "mutable globals")
       .addFeature(FeatureSet::TruncSat, "nontrapping float-to-int operations")
       .addFeature(FeatureSet::SIMD, "SIMD operations and types")
-      .addFeature(FeatureSet::BulkMemory, "bulk memory operations")
+      .addFeature(FeatureSet::BulkMemory,
+                  "bulk memory operations",
+                  FeatureSet(FeatureSet::BulkMemoryOpt))
+      .addFeature(FeatureSet::BulkMemoryOpt,
+                  "memory.copy and memory.fill",
+                  FeatureSet::None,
+                  FeatureSet(FeatureSet::BulkMemoryOpt))
```

Comment:
```suggestion
                  FeatureSet(FeatureSet::BulkMemory))
```
Is this the intended behavior?

Reply:
Er yes, done.